docs/docs/keymaps/behaviors/mouse-emulation.md

```diff
@@ -35,6 +32,15 @@ provided by ZMK near the top:
 #include <dt-bindings/zmk/mouse.h>
 ```
 
+Should you wish to override the default movement or scrolling max velocities, you can define the defaults before including the header, e.g.:
+
+```c
+#define ZMK_MOUSE_DEFAULT_MOVE_VAL 1500
+#define ZMK_MOUSE_DEFAULT_SCRL_VAL 20
```

Comment:
```suggestion
#define ZMK_MOUSE_DEFAULT_MOVE_VAL 1500  // default: 600
#define ZMK_MOUSE_DEFAULT_SCRL_VAL 20    // default: 10
```

I don't know if this is the best way to integrate it into the docs but it'd be good to *mention* what the defaults are so people have a sense of scale when changing them.